corehq/apps/userreports/tasks.py

```diff
@@ -170,7 +170,7 @@ def _run_report(backend_to_use):
             json_response["total_row"] = total_row
         return json_response
 
-    spec = get_document_or_not_found(ReportConfiguration, domain, report_config_id)
```

Comment:
@sravfeyn This code never gets called unless a report has already been grabbed in the main HQ. I hadn't really thought about it before, but seems like enough of an edge case that it's not worth fixing at the moment (especially since I control laboratory UCRs)